From 4d20807ba8c9f3d44ff94313b695dec48345c8be Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Tue, 21 Aug 2018 17:15:36 +0800
Subject: [PATCH 4496/5242] MLK-19258-4 doc: dts: ci-hdrc-usb2: add property
 for picophy

commit  f317c562b65a6ff53464c1c34ce4824a69e9f78f from
https://source.codeaurora.org/external/imx/linux-imx.git

Add two parameters which are used to tune USB signal for picophy,
picophy is used at imx7d and imx845.

Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../devicetree/bindings/usb/ci-hdrc-usb2.txt       |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Documentation/devicetree/bindings/usb/ci-hdrc-usb2.txt b/Documentation/devicetree/bindings/usb/ci-hdrc-usb2.txt
index 56f83b2..dad4b6b 100644
--- a/Documentation/devicetree/bindings/usb/ci-hdrc-usb2.txt
+++ b/Documentation/devicetree/bindings/usb/ci-hdrc-usb2.txt
@@ -106,6 +106,11 @@ i.mx specific properties
 - power-polarity-active-high: add this property if port power function of ehci
   is used to enable vbus, and the vbus power supply chip enable signal is high
   active.
+- picophy,pre-emp-curr-control: picophy is used for imx7d and imx845.
+  HS Treansmitter Pre-Emphasis Current Control may need to tune for USB signal
+  See USBNC_n_PHY_CFG1 for detail.
+- picophy,dc-vol-level-adjust: HS DC Voltage Level Adjustment may need to
+  turn for USB signal, see USBNC_n_PHY_CFG1 for detail.
 
 Examples:
 
-- 
1.7.9.5

